src/pandantic/types.py

```diff
@@ -0,0 +1,8 @@
+from typing import Union
+
+import pandas as pd
+import pydantic
+
+
+SchemaTypes = Union[pydantic.BaseModel]
+DataFrameTypes = Union[pd.DataFrame]
```

Comment:
For now I think "DataFrame" makes sense, but perhaps it's worth considering "Table". Some libraries like `pyarrow` call it a "Table", as well as the database context. 

I could see a world where we define an interface between some sort of Table object for validation, where this could be used beyond in-memory dataframes (i.e., CSV/Parquet files directly, dbase tables, etc.), but that is out-of-scope!

Reply:
I do like the Table naming suggestion. Shall I change it while we are at it currently?